tests/features/shortest_path.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Shortest successful path with all possible information
+
+- [x] Take the shortest path
+- [x] Give most info possible
```

Comment:
Combine these two into the same line, since they seem to correspond to the same scenario?

Reply:
Hmm, were the previous lists always scenarios? I was thinking of them more as features that need testing. Might that be an indication that these each need to be tested separately?

Reply:
I wonder if scenarios list themselves already and whether that's a useful redundancy. On the other hand, it could act as a good checklist of scenarios to write. Then again, it can act as a good checklist of features that need to be tested in the following scenarios.